### DISCUSSION TITLE
Where To Edit Firmware Version & Name To Show In Printer Info

### DISCUSSION THREAD

**Discussion by killlerpenguins** [Jyers/Marlin#257] [Q&A] [answered]
I can't seem to find where to do this at. When in the info section of the printer it shows 3 things. Build size, Firmware Version, & Name. Where do I go to edit the Firmware Version & Name as I would like it to say something else and for the name would like to put the name of my business. Thanks in advance for your help!

![Ender3V2Info](https://user-images.githubusercontent.com/80071719/111069854-65f5a600-84a5-11eb-97a1-ba5ed743340d.jpg)

**Comment by Jyers** [comment, marked as answer]
For contact and build number, in Marlin\src\lcd\extui\lib\creality_dwin\creality_dwin.cpp you can edit CORP_WEBSITE_E and BUILD_NUMBER. For version, in Marlin\src\inc\Version.h you can change SHORT_BUILD_VERSION.

**Reply by killlerpenguins** [reply]
Thanks a ton! I was going crazy trying to search through everything, LOL
